plugins/git/gitlab_statistics

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+# -*- python -*-
+
+"""
+=head1 INTRODUCTION
+
+Plugin to monitor Gitlab status
+
+=head1 INSTALLATION
+
+Usage: Place in /etc/munin/plugins/ (or link it there using ln -s)
+
+=head1 CONFIGURATION
+
+Add this to your /etc/munin/plugin-conf.d/munin-node:
+
+=over 2
+
+    [gitlab]
+    env.logarithmic 1
+    env.hostname gitlab.example.com # required
+    env.token YourPrivateTokenHere # required
+
+=back
+
+=head1 HISTORY
+
+2019-10-02: v 1.0 pcy <pcy.ulyssis.org>: created
+
+=head1 USAGE
+
+Parameters understood:
+
+  config   (required)
+  autoconf (optional - used by munin-config)
+
+=head1 MAGIC MARKERS
+
+#%# family=auto
+#%# capabilities=autoconf
+"""
+
+
+import os
+import json
+import urllib
+import sys
+
+
+def weakbool(x):
+    return x.lower().strip() in {"true", "yes", "y", "1"}
+
+
+url = None
+if 'hostname' in os.environ and 'token' in os.environ:
+    url = "https://" + os.getenv('hostname') \
+        + "/api/v4/application/statistics?private_token=" \
+        + os.getenv('token')
+
+logarithmic = weakbool(os.getenv('logarithmic', 'N'))
+
+
+def reqjson():
+    raw_data = urllib.request.urlopen(url)
```

Comment:
I have to admit, that I was slightly cheating with my short suggestion regarding `urllib`): an additional `except` for `IOError` would be nice in order to deal with occasional network problems. Just printing an error message to stderr and exiting with a non-zero should be sufficient.

Reply:
Done.